sap-successfactors/3.2/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,379 @@
+
+== Use Case: Set Up and Run in Studio
+
+. In Anypoint Studio, click *File > New > Mule Project*.
+. Specify a Project Name and click *Finish*.
+. Search for `http` and drag the HTTP connector to the canvas. 
+. Click the green plus sign next to Connector Configuration, and click *OK* in the menu to accept the default values.
+. Drag the SuccessFactors connector operation to the canvas.
+. Click the green plus next to Connector Configuration, and specify your access credentials.
+. Click a connection strategy for authentication.
+. Configure these properties:
+
+[%header%autowidth.spread]
+|===
+|Parameter|Description
+|CompanyID|Your company ID
+|User Name|User name to connect
+|Password|Password to connect
+|Data Center's Endpoint URL|SuccessFactors Web API URL
+|Enable Session Reuse|The session is alive.
+|===
++
+. Click *OK*.
+
+== To Run a Flow
+
+. In Package Explorer, right click your project's name, and click *Run As > Mule Application*.
+. Check the console to see when the application starts. You should see messages such as these if no errors occur:
+
+[source,text,linenums]
+----
+************************************************************
+INFO  2019-07-14 22:12:42,003 [main] org.mule.module.launcher.DeploymentDirectoryWatcher:
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
++ Mule is up and kicking (every 5000ms)                    +
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+INFO  2019-07-14 22:12:42,006 [main] org.mule.module.launcher.StartupSummaryDeploymentListener:
+**********************************************************
+*  - - + DOMAIN + - -               * - - + STATUS + - - *
+**********************************************************
+* default                           * DEPLOYED           *
+**********************************************************
+
+************************************************************************
+* - - + APPLICATION + - -   * - - + DOMAIN + - -  * - - + STATUS + - - *
+************************************************************************
+* myapp                     * default             * DEPLOYED           *
+************************************************************************
+----
+
+== Example: Success Factors
+
+This example demonstrates the use of SuccessFactors Connector.
+
+To build and run this demo project, you need:
+
+* Anypoint Studio with Mule 4.1.1 or later
+* SuccessFactors Connector v3.0.0 or later
+* SuccessFactors credentials to send OData API requests
+
+=== To Test the Flow
+
+. In Anypoint Studio, in the *File* menu, import the demo project into your workspace.
+. Specify your basic credentials in the `/src/main/app/mule-app.properties` file.
+. Run the project in Studio.
+. Type `localhost:8081` in your browser to access the selection menu of the demo.
+. Optionally, you can configure the following parameters: 
+
+* *Connection Timeout* +
+Timeout for the initial connection with the server
```

Comment:
```suggestion
Timeout for the initial connection with the server.
```

Reply:
No period necessary for sentence fragments